packages/botonic-plugin-contentful/src/cms/exceptions.ts

```diff
@@ -1,12 +1,32 @@
+import { ResourceId } from './callback'
+import { MultiError } from 'async-parallel'
+
 export class CmsException extends Error {
   /**
    * @param message description of the problem
    * @param reason what caused the exception (normally a low level exception)
    */
-  constructor(message: string, readonly reason?: any) {
+  constructor(
+    message: string,
+    readonly reason?: any,
+    readonly resourceId?: ResourceId
+  ) {
     super(CmsException.mergeMessages(message, reason))
   }
 
+  public messageFromReason(): string | undefined {
+    if (!this.reason) {
+      return undefined
+    }
+    if (this.reason instanceof MultiError) {
+      return this.reason.list.map(e => e.message).join('\n')
+    }
+    if (this.reason instanceof Error) {
+      return this.reason.message
+    }
+    return String(this.reason)
+  }
```

Comment:
just a matter of "style", have you considered to save the `{ }` and do the returns in the same lines? Same in other places. Which style guide should we follow in general?

Reply:
there's notable consensus on using braces. See https://softwareengineering.stackexchange.com/questions/16528/single-statement-if-block-braces-or-no